### PR TITLE
#416, #649 SEO friendly URLs

### PR DESCRIPTION
**Pull request by tbl73**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -66,4 +59,28 @@
...
+#    end
+    resources :collection, path: '', only: [:show] do
+#    get 'new_work', on: :member
+#    get 'contributors', on: :member
+      get '/statistics/collection', path: '/statistics', as: :statistics, to: 'statistics#collection'
+
+      match 'display/read_work', path: '/:id', as: :read_work, to: 'display#read_work', via: [:get, :post]
+
+#      get 'transcribe/display_page', path: '/:work_id/:id', as: :transcribe, to: 'transcribe#display_page'
+    end
```
Comment (saracarl) [inline config/routes.rb]: clean up the commented out stuff.

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -211,6 +222,7 @@ def load_html_blocks
   def authorize_collection
     # skip irrelevant cases
     return unless @collection
+    return if @collection.is_a?(DocumentSet)
```
Comment (tbl73) [inline app/controllers/application_controller.rb]: Question about rewriting line 228 - would that affect access if someone is using an old URL to access the document set?

**Comment by tbl73** [comment]
It turns out that notes also include a work_id, so I was able to limit the document_sets.notes method to notes that are in the works from the document set.

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (saracarl) [inline app/controllers/application_controller.rb]: We're willing to break existing document set URLs (so few -- or no -- users), so don't worry about this.

Comment (tbl73) [inline app/controllers/application_controller.rb]: Okay, just pushed that change up.

File: app/views/work/edit.html.slim [hunk 3]
```diff
@@ -17,13 +17,13 @@
...
+          td =f.text_field :slug, value: @work.slug
+        tr
           th
           td 
-            p <b>Current URL:</b> The current URL for this work is <b>www.fromthepage.com/#{@collection.owner.slug}/#{@collection.slug}/#{@work.slug}</b>.  If you want to edit the work section of the URL, please use lowercase letters and dashes between any words.
-        tr
-          th =f.label :slug, 'Edit URL'
-          td =f.text_field :slug, value: @work.slug
-
+            |The current URL for this work is <b>www.fromthepage.com/#{@collection.owner.slug}/#{@collection.slug}/#{@work.slug}</b>.  If you want to edit the work section of the URL, please use lowercase letters and dashes between any words.
```
Comment (saracarl) [inline app/views/work/edit.html.slim]: here too.

**Pull request merged**